config.mk

```diff
@@ -130,7 +130,7 @@ thread_model=kos
 # Use 'install-strip' mode for removing debugging symbols of the toolchains.
 # Use 'install' only if you want to enable debug symbols for the toolchains.
 # This may be useful only if you plan to debug the toolchain itself.
-install_mode=install-strip
+install_mode=install
```

Comment:
Why re-enabling full `install` including debugging symbols for the toolchains? Any reason for this?

Reply:
The intention was to debug the gcc-10.1.0-kos.diff code migration.